client/packages/common/src/types/exceptions.ts

```diff
@@ -0,0 +1,20 @@
+import { AuthError } from '../authentication';
+
+interface Extensions {
+  details: string;
+}
+
+interface Location {
+  column: number;
+  line: number;
+}
+
+interface ApiException {
+  extensions: Extensions;
+  locations: Location[];
+  message: string;
+  path: string[];
+}
+
+export const isPermissionDeniedException = (e?: any) =>
```

Comment:
I couldn't quite trace this, when is this actually used ? StatusPanel doesn't get called with error prop

Reply:
Sorry I see it now, go to references in vs code didn't quite work : /